### DISCUSSION TITLE
Change interface language

### DISCUSSION THREAD

**Discussion by vanbang9710** [xushengfeng/eSearch#218] [Q&A] [answered]
I don't know Chinese so I don't know how to change interface language to English!

**Comment by xushengfeng** [comment, marked as answer]
Right-click on the tray, find "设置", search for "English"

**Reply by vanbang9710** [reply]
Where is "English"? ![|10pct](https://i.imgur.com/Vd9r3hI.png)

**Reply by xushengfeng** [reply]
click top right bar，input "English"
